Userland/Libraries/LibVideo/VP9/BitStream.h

```diff
@@ -38,11 +38,11 @@ class BitStream {
     size_t range_coding_bits_remaining();
 
     /* (4.9.2) */
-    ErrorOr<i8> read_s(size_t bits);
+    ErrorOr<i16> read_s(size_t bits);
 
     u64 get_position();
-    size_t bytes_remaining();
     size_t bits_remaining();
+    size_t bytes_remaining();
```

Comment:
This change seems unnecessary? It just moves the definition 2 lines downward.

Reply:
Ah, this was just to match the order in the cpp file so that it makes sense since one references the other. I'll revert this.